mlir/lib/Conversion/MemRefToEmitC/MemRefToEmitCPass.cpp

```diff
@@ -50,6 +57,31 @@ struct ConvertMemRefToEmitCPass
     if (failed(applyPartialConversion(getOperation(), target,
                                       std::move(patterns))))
       return signalPassFailure();
+
+    mlir::ModuleOp module = getOperation();
+    module.walk([&](mlir::emitc::CallOpaqueOp callOp) {
+      if (callOp.getCallee() != "aligned_alloc") {
+        return mlir::WalkResult::advance();
+      }
+
+      for (auto &op : *module.getBody()) {
+        if (auto includeOp = llvm::dyn_cast<mlir::emitc::IncludeOp>(op)) {
+          if (includeOp.getIsStandardInclude() &&
+              ((options.lowerToCpp && includeOp.getInclude() == "cstdlib") ||
+               (!options.lowerToCpp && includeOp.getInclude() == "stdlib.h"))) {
+            return mlir::WalkResult::interrupt();
+          }
```

Comment:
You can reduce the nesting here by using `continue` if the dyn_cast fails. Then you won't need to brace around the rest of the loop.